### ISSUE TITLE
Argument Error Raised Calling Reduced View

### ISSUE THREAD

**Issue by JamesHayton** [couchrest/couchrest_model#14]
I just upgraded my rails 3 site to use couchrest_model to take advantage of the active model goodness and for the most part things work as expected.  Now I can get rid of my hacked together active_model compliant couchrest extended document.  Thanks!

I did however run into a problem:

I have a view in my model that worked perfectly with couchrest extended document, but doesn't work any longer with couchrest model. 

The view just emits a complex key and then 1 and I have a reduce function that sums up the view.

When I call the view in my controller like this:

@closed = Trade.by_system(:key => [@system.id, "Closed"], :descending => true)

Extended Document would automatically have this call the non-reduced view unless you specified :reduce => true.

So, I thought to myself, well why don't I just call :reduce => false, but that didn't work either. 

The long story short is that views with reduce functions can only be accessed with :reduce => true and there is no way to get the non reduced output any more.  I get an argument error when trying to call the non reduced version. 

Unless I am missing something, I think this is a bug.

Bundling From Git
Rails 3.0.0
Latest Couchrest Gem
